src/IkBot.py

```diff
@@ -303,13 +320,11 @@ async def parse():
 
                 # Self Death
                 elif 'SelfDeath' in event[0]:
-                    taunt_death = taunt_Sheet.range('B:B', returnas='matrix')
-                    await client.alarm(f'{elf.char_name} has fallen to {event[1]}! {random.choice(taunt_death)[0]}')
+                    await client.alarm(f'{elf.char_name} has fallen to {event[1]}! {random.choice(taunt_death_list)}')
```

Comment:
I am not entirely sure this will work as expected. `taunt_death` is a 2x2? matrix i think while `taunt_death_list` is just a list. I may need to create my own google sheet to play around more with getting tables from google sheets